src/adaptermanager.js

```diff
@@ -113,7 +113,7 @@ function getAdUnitCopyForPrebidServer(adUnits) {
   let adUnitsCopy = utils.deepClone(adUnits);
 
   adUnitsCopy.forEach((adUnit) => {
-    adUnit.sizes = transformHeightWidth(adUnit);
+    adUnit.sizesS2S = transformHeightWidth(adUnit);
```

Comment:
can we just move this function into prebidServerBidAdapter instead of adding a temporary sizesS2S?

Reply:
Yep - I'll move the function over and make other adjustments.